sdk/servicebus/service-bus/src/util/atomError.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { Constants as CoreHttpConstants, HttpOperationResponse } from "@azure/core-http";
+import { Constants } from "./utils/constants";
+
+/**
+ * Represents custom `Error` information returned by ATOM based services.
+ */
+export class AtomError extends Error {
```

Comment:
Should this extend `RestError`?

Reply:
Oh, this file should have been deleted since we decided to re-use `RestError` - updated PR